src/main/java/com/thebois/controllers/gamecontrollers/ColonyController.java

```diff
@@ -0,0 +1,39 @@
+package com.thebois.controllers.gamecontrollers;
+
+import com.thebois.controllers.IController;
+import com.thebois.models.world.World;
+import com.thebois.views.gameviews.IView;
+import com.thebois.views.gameviews.ColonyView;
+
+/**
+ * Collects data form the world and sends it to its corresponding view.
+ */
+public class ColonyController implements IController<IView> {
+
+    private final World world;
+    private final ColonyView colonyView;
+
+    /**
+     * Creates a colony controller.
+     *
+     * @param world    The world that controller should get data from.
+     * @param tileSize The tile size for the tiles in the world.
+     */
+    public ColonyController(final World world, final float tileSize) {
```

Comment:
I didn't really touch the constructor of the controllers expect that they instantiate their own view, but what you say is resonable so I can make that change if we want to.

Reply:
I see. Yeah we're way past the point of avoiding merge conflicts since you've already made changes to the constructor, might as well make it pretty. :)